### PR TITLE
Update API Security pages

### PR DESCRIPTION
**Pull request by isharailanga**
This PR is to add content according to https://docs.google.com/document/d/16XzPb4Nxb5RJ77dwIgfrAmA7S1S3HZoZPOLe2fbTcMw/edit.

### CODE REVIEW

File: en/docs/learn/api-security.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ This validation allows banks to validate TPPs from the NCAs. This is done by
 validating the transport layer certificate a TPP has obtained. 
 
 ###Role validation 
-According to the [Open Banking Standard - UK](https://www.openbanking.org.uk/about-us/glossary/), TPPs have roles 
+According to the NextGenPSD2 specification, TPPs have roles 
 for the services they provide. The TPP’s role is defined in their transport certificate (QWAC certificate 
 that they obtain from an NCA), which WSO2 Open Banking checks in the API and Dynamic Client Registration flows.
```
Comment (Akila94) [inline en/docs/learn/api-security.md]: Remove "Dynamic Client Registration" part.

Comment (Akila94) [inline en/docs/learn/api-security.md]: Remove "Dynamic Client Registration" part.

**Review by Akila94 — APPROVED** [review]

File: en/docs/learn/api-security.md (hunk 1 above)
Comment (isharailanga) [reply en/docs/learn/api-security.md]: Fixed in 11987f2244d0bebd44fc743bac5051c8e803e7e7

Comment (isharailanga) [reply en/docs/learn/api-security.md]: Fixed in 11987f2244d0bebd44fc743bac5051c8e803e7e7

**Pull request merged**